prisma/seedDev.mjs

```diff
@@ -235,22 +240,26 @@ async function main() {
     });
   }
   for (let i = 0; i < 10; i += 1) {
-    const organizationData = randomOrganization({ therapies, districts, organizationTypes });
+    const organizationData = randomOrganization({
+      therapies,
+      districts,
+      organizationTypes,
+      expertSpecializations: specializations,
+    });
     // eslint-disable-next-line no-await-in-loop
-    await prisma.$transaction(async trx => {
-      const organization = await trx.organization.create({
-        data: organizationData,
-      });
-      await trx.searchEntry.create({
-        data: {
-          sortString: organization.name,
-          organization: {
-            connect: {
-              id: organization.id,
-            },
+    const organization = await prisma.organization.create({
+      data: organizationData,
+    });
+    // eslint-disable-next-line no-await-in-loop
+    await prisma.searchEntry.create({
+      data: {
+        sortString: organization.name,
+        organization: {
+          connect: {
+            id: organization.id,
```

Comment:
Since this is seed that's not super-important, but you could do this better:
1) Use `createMany` to decrease amount of DB-calls
2) Nest `searchEntry` creation into `organization` creation
3) Omit for-loop in favor of `map`

Reply:
thanks for suggestions!
1. unfortunately prisma createmany do not support connections
2. that is cool idea, i will try to do that
3. with `map` we would need to use neither `promise.all`  ( which sometimes exeeds our allowed pool size for db connections on our trial db't 😅 ) or `reduce` to build these calls into one chain ( which will look a bit overcomplicated comparing to plain old `for` loop ), from my side i think `for` is KISS way of doing things, which looks alright for seeds...how do you think?

Reply:
2. done, thanks!